File: program/program/calculate-compound-interest/calculateCompoundInterest.scala
import scala.io.StdIn._;

object compoundInterest {
  def main(args:Array[String]):Unit={
    val principal = readLine().toInt;
    println("Enter the principal amount : "+principal);
    val interest = readLine().toInt;
    println("Enter the rate of interest : "+interest);
    val time = readLine().toInt;
    println("Enter the time : "+time);
    val compoundInterest = principal * Math.pow((1 + interest), time) ;
    println("Compound Interest :"+compoundInterest);
  }
}

File: program/program/calculate-compound-interest/CalculateCompoundInterest.c


#include <stdio.h>
#include <math.h>
int main()
{
    float CI, rate, PA, time;
    printf("Enter principal amount :");
    scanf("%f", &PA);
    printf("Enter the rate :");
    scanf("%f", &rate);
    printf("Enter time :");
    scanf("%f", &time);
    // formula to calculate compound interest
    CI = PA * (pow((1 + rate / 100), time));

    printf("Compound interest : %0.2f\n", CI);
    return 0;
}